lib/census/client.rb

```diff
@@ -89,6 +89,15 @@ def get_users(limit: 100, offset: 0)
       end
     end
 
+    def get_users_by_cohort_name(cohort_name)
+      users_search_url = build_full_url_with_token(path: "/api/v1/users/search_all") + "&1=#{cohort_name}"
```

Comment:
Looking at https://github.com/turingschool-projects/census/blob/47917577612677deb34ad63899274fb1194ceb7e/app/controllers/api/v1/users/search_all_controller.rb#L6 I think this param currently written as `1` should really be a `q`. No?